templates/sysconfig.erb

```diff
@@ -0,0 +1,15 @@
+# This file is managed by Puppet
+
+<%- if !@java_home.nil? && !@java_home.empty? -%>
+JAVA_HOME=<%= @java_home %>
+<%- end -%>
+
+JAVA_OPTS=<%= @java_opts %> <%= @java_extra_opts %>
+
+<% @env_var.sort.each do |key,val| -%>
+<%= key %>=<%= val %>
+<% end if @env_var -%>
+
+<% @all_options.each do |k,v| -%>
+MARATHON_<%= k.upcase %>=<%= v %>
```

Comment:
Is there any reason why you want to have all the config options here? I'd rather we provided the choice to the user of the module.